go/internal/services/keys/get_migrated.go

```
@@ -0,0 +1,138 @@
+		newHash := hash.Sha256(rawKey)
+		err = db.Query.UpdateKeyHashAndMigration(ctx, s.db.RW(), db.UpdateKeyHashAndMigrationParams{
+			ID:                 key.Key.ID,
+			Hash:               newHash,
+			Start:              extractStart(rawKey),
+			PendingMigrationID: sql.NullString{Valid: false, String: ""},
+			UpdatedAtM:         sql.NullInt64{Valid: true, Int64: time.Now().UnixMilli()},
+		})
+		if err != nil {
+			return nil, log, fault.Wrap(
+				err,
+				fault.Code(codes.URN(codes.UnkeyAppErrorsInternalServiceUnavailable)),
```

golangci-lint (2.2.2)

unnecessary conversion

(unconvert)